src/familybot/lib/database.py

```
@@ -19,199 +22,256 @@
+                        logger.info(
+                            f"Database: Adding column '{column}' to table '{table}'."
                         )
-                        if update_val is not None:
+                        try:
                             cursor.execute(
-                                f"UPDATE {table} SET {column} = {update_val} WHERE {column} IS NULL"
+                                f"ALTER TABLE {table} ADD COLUMN {column} {definition}"
+                            )
+                            if update_val is not None:
+                                cursor.execute(
+                                    f"UPDATE {table} SET {column} = {update_val} WHERE {column} IS NULL"
```

Ruff (0.15.7)

[error] Possible SQL injection vector through string-based query construction

(S608)